CHANGELOG.md

```
@@ -815,24 +799,40 @@ for previous changelogs._
 <!-- PR links -->
 
 [#2157]: https://github.com/munich-quantum-toolkit/core/pull/2157
+
 [#2156]: https://github.com/munich-quantum-toolkit/core/pull/2156
```

markdownlint-cli2 (0.23.2)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "`#2156`"

(MD053, link-image-reference-definitions)